scripts/build-breakpad.sh

```diff
@@ -4,42 +4,39 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# Jenkins build script for building Breakpad
+# Build script for building Breakpad
+# Generally run in Taskcluster, but split out to a separate script
+# so it can be run for local builds if necessary without assuming
+# the Taskcluster environment.
 
 # any failures in this script should cause the build to fail
-set -e
+set -v -e -x
+
+export MAKEFLAGS=-j`grep -c ^processor /proc/cpuinfo`
+
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+export PATH=`pwd`/depot_tools:$PATH
 
 # Checkout and build Breakpad
 echo "PREFIX: ${PREFIX:=`pwd`/build/breakpad}"
-svn co http://google-breakpad.googlecode.com/svn/trunk google-breakpad
-cd google-breakpad
+mkdir breakpad
+cd breakpad
+fetch breakpad
+cd src
 mkdir -p ${PREFIX}
-rsync -a --exclude="*.svn" ./src ${PREFIX}/
+rsync -a --exclude="*.git" --exclude="*.svn" ./src ${PREFIX}/
```

Comment:
I *think* when I originally wrote this they still had some deps in SVN but they're all git now.